todo_app/app.py

```diff
@@ -48,12 +89,15 @@ def add_card():
         post = cards.insert_one({"name": card_name, "status": "To Do", "dateLastActivity": datetime.now()})
         return redirect(url_for('get_cards'))
 
-
+    @writer_required
     @app.route('/items/<_id>', methods = ['POST'])
```

Comment:
If you test this with a Github account that shouldn't have writer permissions, you'll find that this is currently not running 😞 Flask is fussy with it's decorators, the route decorator needs to be the outermost one (i.e. the first one) otherwise it doesn't record the other decorators & they don't get executed - so you'll need to reorder these. I'd also suggest adding in the `@login_required` decorator as well as the `@writer_required` - it'll help ensure you have clearer distinct behaviour in the different cases that arise
```suggestion
    @app.route('/items/<_id>', methods = ['POST'])
    @writer_required
```

Reply:
Done